### ISSUE TITLE
closeMarket fails during reporting sequence (sometimes)

### ISSUE THREAD

**Issue by tinybike** [AugurProject/augur.js#128]
During test/reporting/sequence.js, occasionally closeMarket will fail with the following error:
```javascript
{ error: '0',
  message: 'fail/trading not over yet/event not expired or closed already' }
```

**Comment by tinybike** [comment]
closeMarket seems to be during invoked during the correct period (+2 from event expiration):
```
 - Vote period:           2448756
 - Expiration period:     2448756
 - Current period:        2448758
 - Last period:           2448755
 - Last period penalized: 2448755
 - Rep redistribution:    1
 - Residual: 22/600 (3.6666666666666665%)
```

**Comment by tinybike** [comment]
When this happens, during the period after the event's expiration, the call to `penalizeWrong` will have failed with this error code:
```
penalizeWrong(branch, 0) error: { error: '-8',
  message: 'needed to collect fees last period which sets the before/after rep' }
```

**Comment by tinybike** [comment]
Fixed in cde108ea3428981ff46e65b280b5e44e7dc7ed2c
